clean_basin_data.r

```diff
@@ -25,22 +25,37 @@ names(basin) <- str_to_lower(names(basin))
 stations <- read_csv(file = "Data/waterbody_stations_20101104.csv", col_names = T)
 kasky_stations <- stations %>% filter(PU_CODE == "kasky")
 names(kasky_stations) <- str_to_lower(names(kasky_stations))
+kasky_stations<- rename(kasky_stations, station_code = station_co)
 
-## Convert basin "Waterbody Station" to readable format. 
-basin_stations <- unique(basin$`Waterbody Station`) %>% tibble()
-basin_stations <- rename(basin_stations, Waterbody_Station = .)
-basin_stations$Station_Name <- stringr::str_extract(basin_stations$Waterbody_Station, ".*(?=:)")
-basin_stations$Waterbody_Location <- stringr::str_extract(basin_stations$Waterbody_Station, "(?<=:).*")
+### Convert basin "Waterbody Station" to readable format. 
+# basin_stations <- unique(basin$waterbody_station) %>% tibble()
+# basin_stations <- rename(basin_stations, waterbody_station = .)
+# basin_stations$station_name <- stringr::str_extract(basin_stations$waterbody_station, ".*(?=:)")
+# basin_stations$waterbody_location <- stringr::str_extract(basin_stations$waterbody_station, "(?<=:).*")
+# 
+# basin_stations$waterbody <- unique(basin$waterbody)
+# basin_stations$dupe <- stringr::str_extract(basin_stations$waterbody)
```

Comment:
Since this is version controlled you don't need to have all of these comments here. I would recommend removing them to keep your code clean and readable.

Reply:
Thank you for your suggestion.

Reply:
I made the changes you suggested and now my code is cleaner 👯